infrastructure/metadata/infrastructure/expected-fail/window-onload-test.html.ini

```diff
@@ -0,0 +1,17 @@
+[window-onload-test.html]
+  expected: OK
```

Comment:
We can leave out `expected: OK` because it's the default.

Reply:
updated